optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/employee/EmployeeRestService.java

```diff
@@ -88,58 +86,59 @@ Boolean removeEmployee(@ApiParam(required = true) @PathParam("tenantId") Integer
     @POST
     @Path("/availability/add")
```

Comment:
Similarly, I think the idiomatic thing for this method would be a POST against "/tenant/{tenantId}/employee/availability/".